### PR TITLE
Check for WER UI

### PR DESCRIPTION
**Pull request by RanjanarMS**

### CODE REVIEW

File: Tasks/DeployVisualStudioTestAgent/SetupTestMachineForUITests.ps1 [hunk 1]
```diff
@@ -49,6 +49,52 @@ function IsAutoLogonDisabled()
...
+    if ([string]::IsNullOrEmpty($dontShowUI))
+    {
+        Write-Verbose -Message "Registry path $registryPath found. DontShowUI key is not set." -Verbose
+        return $false
+    }
+    elseif($dontShowUI -eq "1")
+    {
+        return $true
+    }    
+     elseif($dontShowUI -eq "0")
```
Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/SetupTestMachineForUITests.ps1]: fix indention

File: Tasks/DeployVisualStudioTestAgent/SetupTestMachineForUITests.ps1 [hunk 2]
```diff
@@ -49,6 +49,52 @@ function IsAutoLogonDisabled()
...
+        return $false
+    }
+}
+
+function IsWindowsErrorReportingDontShowUISet($TestUserDomain, $TestUserName)
+{  
+    if( -not(IsDontShowUISetInRegistryPath -registryPath "HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting"))
+    {
+        $filter = "name = '" + $TestUserName +"' AND domain = '" + $TestUserDomain + "'" 
+        $user = Get-WmiObject win32_useraccount -Filter $filter
```
Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/SetupTestMachineForUITests.ps1]: Ignore SIlent Action

Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/SetupTestMachineForUITests.ps1]: or wrap around try catch so that ignore on error

Comment (RanjanarMS) [inline Tasks/DeployVisualStudioTestAgent/SetupTestMachineForUITests.ps1]: Done

**Pull request merged**